java-generator/gradle-plugin/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>java-generator-parent</artifactId>
+    <groupId>io.fabric8</groupId>
+    <version>6.6-SNAPSHOT</version>
+  </parent>
+
+  <!--
+          GAV has to be compliant with plugin id
+          https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers
+          ${plugin.id}:${plugin.id}.gradle.plugin:${version}
+   -->
+  <groupId>io.fabric8.java-generator</groupId>
+  <artifactId>io.fabric8.java-generator.gradle.plugin</artifactId>
+  <name>Fabric8 :: Java generator :: Gradle Plugin</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>java-generator-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.codehaus.groovy</groupId>
+      <artifactId>groovy-all</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.gradle</groupId>
+      <artifactId>gradle-base-services</artifactId>
```

Comment:
Forgot to mention, but I think that most of these dependencies should be marked as `provided`.

Reply:
Yep, I noticed that is the case in Jkube. I'll have a look again and fix it, thanks!